### ISSUE TITLE
It's not really correct to call all registered members "contributors."

### ISSUE THREAD

**Issue by ckjpn** [Tatoeba/tatoeba2#3047]
This could be considered a bug, I suppose, but I'll add it as an enhancement request.

### The Bug

It's not really correct to call all registered members "contributors."

https://tatoeba.org/en/users/all?sort=since&direction=desc

We have many registered members who have made no contributions.

### The Possible Enhancement
I wonder how difficult it would be to have another status for registered members who have not yet contributed anything, and then automatically changing that status to "contributor" is they add anything to the website -- a sentence, a comment, an audio file, a wall posting.

Possible names for the status -- just brainstorming.

registered member
non-contributing member
potential contributor
member with no contributions

Likely someone else can come up with a better name.

**Comment by vinkaks** [comment]
Would be interested in implementing it if the idea is developed further (discussed on wall maybe?) and need is high..

**Comment by DJ-Saidez** [comment]
I think either just "member", or leaving it as contributor would be best. Having the extra words with member seems unneeded and space-occupying, and possibly if new users see contributors by their username when they register, it might encourage them, even if slightly, to contribute, so as to fit what they were called.

**Comment by jiru** [comment]
I agree with DJ-Saidez’s analysis. Contributor is referring to the role/status of the member, rather than a statement about what they actually do. Aren’t there some admins not actually taking on their role of admin on Tatoeba? Should we revoke their status on that basis? No. (Okay I admit that was a bit far-fetched :stuck_out_tongue_winking_eye:)

That being said, I can see that CK would like to have a way to discriminate "non-contributing contributors". I, too, think it’s a good idea because it would allow giving a much better view of how many members are involved in the project, especially among all the "spam accounts" (see #1613). I just don’t think the proposed solution is appropriate.

@ckjpn Can you elaborate on the reasons you are asking for that? (I am asking because you may have different reasons that what I mentioned.)

**Comment by ckjpn** [comment]
Primarily because we have a lot of spam accounts, and also a lot of accounts of people who likely never return to the website.

I suspect a lot of people register for an account thinking that they need one, but then realize they can do what they want to do on the website without having an account.

It seems sort of dishonest to say we have over 60,000 contributors.

**Comment by jiru** [comment]
> It seems sort of dishonest to say we have over 60,000 contributors.

If you are referring to the title of [this page](https://tatoeba.org/fr/users/all), it rather says that we have over 60,000 _members_, which is not exactly the same thing. To my knowledge, there is nowhere on the site we can actually see how many members have the contributor role in total. Could you point out what information on which page seems dishonest to you?

> I suspect a lot of people register for an account thinking that they need one, but then realize they can do what they want to do on the website without having an account.

I don’t know how many of such accounts exist, but I’d rather not see them as a nuisance or something we wanna exclude in order to adjust a statistic.  Instead, I wish we can better represent them, I mean in a more positive way than anything related to their lack of contributions. I think some people may have signed up to Tatoeba simply because they like the project. Some may have only signed up in order to report an error on a sentence. Some may only be creating lists or asking for vocabulary.
